packages/shared/src/contexts/AuthContext.tsx

```diff
@@ -138,7 +138,7 @@ export const AuthContextProvider = ({
   const referral = user?.referralId || user?.referrer;
   const referralOrigin = user?.referralOrigin;
 
-  if (firstLoad === true && endUser && !endUser?.infoConfirmed) {
+  if (isRegistering && endUser && !endUser?.infoConfirmed) {
```

Comment:
Why do we log people out if they registering?

Reply:
Ah, it is actually the opposite, wrong naming here. If they are not registering, and info is not confirmed, we will log them out.